source/_templates/layout.html

```diff
@@ -8,7 +8,6 @@
 <!-- Page content header -->
 {% block breadcrumbs %}
   <ul class="buttons-header">
-    <li><a class="button-appearance" href="https://media.readthedocs.org/pdf/openfisca/latest/openfisca.pdf"><i class="fas fa-download"></i> Documentation</a>
```

Comment:
> Great! Did you check if we have some task trying to generate the PDF somewhere? It might be worth switching it off if that's the case 🙂

Reply:
Looking through read the docs, it seems to me that it is managed by them.